sagemaker-python-sdk/tensorflow_serving_container/tensorflow_serving_container.ipynb

```diff
@@ -0,0 +1,343 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using the SageMaker TensorFlow Serving Container\n",
+    "\n",
+    "The [SageMaker TensorFlow Serving Container](https://github.com/aws/sagemaker-tensorflow-serving-container) makes it easy to deploy trained TensorFlow models to a SageMaker Endpoint without the need for any custom model loading or inference code.\n",
+    "\n",
+    "In this example, we will show how deploy one or more pre-trained models from [TensorFlow Hub](https://www.tensorflow.org/hub/) to a SageMaker Endpoint using the [SageMaker Python SDK](https://github.com/aws/sagemaker-python-sdk), and then use the model(s) to perform inference requests."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "First, we need to ensure we have an up-to-date version of the SageMaker Python SDK, and install a few\n",
+    "additional python packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -U --quiet \"sagemaker>=1.14.2\"\n",
+    "!pip install -U --quiet opencv-python tensorflow-hub"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we'll get the IAM execution role from our notebook environment, so that SageMaker can access resources in your AWS account later in the example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker import get_execution_role\n",
+    "\n",
+    "sagemaker_role = get_execution_role()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download and prepare a model from TensorFlow Hub\n",
+    "\n",
+    "The TensorFlow Serving Container works with any model stored in TensorFlow's [SavedModel format](https://www.tensorflow.org/guide/saved_model). Normally, these would be the output of your own\n",
```

Comment:
are there legitimate use cases for using a pre-trained model? if so, I'd probably change "Normally, these would be the output of your own training jobs" to something like "This can be the output of your own training job or a model pre-trained elsewhere"

Reply:
good idea